spec/spec.md

```diff
@@ -1176,6 +1175,28 @@ When using this [[ref:Feature]]:
   property. If present, its value ****MUST**** be a
   [JSON LD Framing Document](https://w3c.github.io/json-ld-framing/) object.
 
+### Retention Feature
+
+``The Retention [[ref:Feature]] extends the [[ref:Input Descriptor Object]]'s _field_ 
+object, allowing a Verifier to indicate it will retain the submitted value for the
+specific field.  It is currently presented to support mDL systems, and may be deprecated in the
+ future or a separate specification for more robust mDL interoperability is created at a later time.
+
+A [[ref:Verifier]] must not retain any data, including digests, signatures,
+and derived data received, except for claims from the [[ref:Presentation Submission]] descriptors
+for which the accompanying [[ref:Input Descriptor Object]]'s _field_ `intent_to_retain`
+property was set to true.
+
+Retain is defined as: “to store for a period longer than necessary to conduct the
+exchange between [[ref:Holder]] and [[ref:Verifier]] in realtime”.
```

Comment:
What is meant by "the exchange between ..."? Is that the presentation exchange process?
Does that mean the verifier can't keep the data longer than the exchange period, and once they get it they have to immediately throw it away?
If that is the case, why are they asking for the data if they need to throw it away as soon as they get it?

There's something I'm not understanding here.